src/vmshepherd/presets/preset.py

```diff
@@ -83,36 +83,41 @@ def _reset_counters(self):
         """
         if if_older_than is None or time.time() - self._vms_refresh_time > if_older_than:
```

Comment:
Since `if_older_than` looks like the boolean flag I would suggest to actually use boolean values (`True` and `False`). IMO using `None` for that purpose is not the greatest idea - `None` does not provide any information about its state.

Reply:
You are right, the logic was changed.